command/agent/job_endpoint.go

```diff
@@ -1385,6 +1390,14 @@ func ApiCSIPluginConfigToStructsCSIPluginConfig(apiConfig *api.TaskCSIPluginConf
 	return sc
 }
 
+func ApiActionToStructsAction(job *structs.Job, action *api.Action, act *structs.Action) *structs.Action {
```

Comment:
```suggestion
func ApiActionToStructsAction(job *structs.Job, action *api.Action) *structs.Action {
```

Reply:
Oh! Surprised not to get an "unused param" error here or something. Good catch.